SPECS/python3-cassandra-driver/python3-cassandra-driver.spec

```diff
@@ -0,0 +1,84 @@
+Summary:        A modern, feature-rich and highly-tunable Python client library for Apache Cassandra (2.1+)
+Name:           python3-cassandra-driver
+Version:        3.24.0
```

Comment:
This version is a couple of years old by this point. Can we upgrade to 3.25.0?

Reply:
Upgraded